README.md

```diff
@@ -1,2 +1,66 @@
-# firstCapstone
-this is first capstone project in Microverse
+# Space Lovers project
+
+"Space exploration is the use of astronomy and space technology to
+ explore outer space. While the exploration of space is carried out
+ mainly by astronomers with telescopes, its physical exploration
+ though is conducted both by unmanned robotic space probes and human
+ spaceflight"
+
+
+![Alt text](/assets/sc1.png)
+![Alt text](/assets/sc2.png)
+![Alt text](/assets/sc3.png)
+![Alt text](/assets/sc4.png)
+
+
+## Built With
+
+
+- HTML, CSS
+- stylelint
+- Eslint
+
+
+## Live Demo
+
+https://realiens.github.io/firstCapstone/
+
+
+## presentation for app
+https://www.loom.com/share/e129bd52bf454ef1ba81810a7c1e4d00
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Live server on my local machine
+
+### Setup
+
+Open with live server on vscode
+
+### Install
+no installations
+
+
+## Deployment
+ 
+Deployed with github pages as a host
+
+
+## Authors
+
+👤 **Ahmed Reda**
+
+- GitHub: [@githubhandle](https://github.com/ReAliens)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Please add "Acknowledgements" section that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).